src/LogDensityProblems.jl

```diff
@@ -159,6 +159,8 @@ struct TransformedLogDensity{T <: TransformVariables.AbstractTransform, L}
     log_density_function::L
 end
 
+Base.@deprecate_moved TransformedLogDensity "TransformedLogDensities.TransformedLogDensity"
```

Comment:
I think one should just state the package name here:
```suggestion
Base.@deprecate_moved TransformedLogDensity "TransformedLogDensities"
```

If I understand correctly, one should also remove all occurrences of `TransformedLogDensity` (and probably all traits for it) since otherwise the deprecation doesn't show up but is "overridden" by the type definition. Users will then see a deprecation warning that points them towards installing and loading `TransformedLogDensities`.

The macro also exports `TransformedLogDensity` automatically, so it could be removed from the exports. Similarly, I think the dependency on TransformVariables could be removed.

Reply:
Good point, doing that in #89 then.